WooCommerce/Classes/ViewRelated/Products/Edit Product/Shipping Settings/Product+ShippingSettingsViewModels.swift

```diff
@@ -0,0 +1,48 @@
+import Yosemite
+
+extension Product {
+    func createShippingWeightCellViewModel(shippingSettingsService: ShippingSettingsService,
```

Comment:
Suggestion: `Cell` is implied by the location of where it's used. You could delete that word in the function name to shorten it. And "shippingSettingsService" could have parameter naming like this: `using shippingSettingsService` to improve the readability. 

``` func createShippingWeightViewModel(using shippingSettingsService: ShippingSettingsService,```